deploy/install-driver-standalone.sh

```diff
@@ -33,8 +33,15 @@ kubectl apply -f $repo/csi-azuredisk-controller.yaml
 kubectl apply -f $repo/csi-azuredisk-node.yaml
 
 if [[ "$#" -gt 1 ]]; then
-  if [[ "$2" = "windows" ]]; then
-    echo "Install Azure Disk CSI Windows driver ..."
+  if [[ "$2" == *"snapshot"* ]]; then
```

Comment:
@ZeroMagic we also need to add snapshot/windows driver selection in helm chart installation part. Could you work on that? Thanks.

Reply:
will do it later.